scenes/animations.ts

```diff
@@ -356,4 +356,436 @@ function createSkeletonAnimations(scene: Phaser.Scene) {
+    })
+
+    scene.anims.create({
+        key: male1 + '-idle-right',
+        frames: [{ key: male1, frame: 7 }]
+    })
+
+    //male 2
+    scene.anims.create({
+        key: male2 + '-idle-up',
+        frames: [{ key: male2, frame: 10 }]
+    })
+
+    scene.anims.create({
+        key: male2 + '-idle-down',
+        frames: [{ key: male2, frame: 1 }]
+    })
+
+    scene.anims.create({
+        key: male2 + '-idle-left',
+        frames: [{ key: male2, frame: 4 }]
+    })
+
+    scene.anims.create({
+        key: male2 + '-idle-right',
+        frames: [{ key: male2, frame: 7 }]
+    })
+
+    //male 3
+    scene.anims.create({
+        key: male3 + '-idle-up',
+        frames: [{ key: male3, frame: 10 }]
+    })
+
+    scene.anims.create({
+        key: male3 + '-idle-down',
+        frames: [{ key: male3, frame: 1 }]
+    })
+
+    scene.anims.create({
+        key: male3 + '-idle-left',
+        frames: [{ key: male3, frame: 4 }]
+    })
+
+    scene.anims.create({
+        key: male3 + '-idle-right',
+        frames: [{ key: male3, frame: 7 }]
+    })
+
+    //male 4
+    scene.anims.create({
+        key: male4 + '-idle-up',
+        frames: [{ key: male4, frame: 10 }]
+    })
+
+    scene.anims.create({
+        key: male4 + '-idle-down',
+        frames: [{ key: male4, frame: 1 }]
+    })
+
+    scene.anims.create({
+        key: male4 + '-idle-left',
+        frames: [{ key: male4, frame: 4 }]
+    })
+
+    scene.anims.create({
+        key: male4 + '-idle-right',
+        frames: [{ key: male4, frame: 7 }]
+    })
+
+    //male 5
+    scene.anims.create({
+        key: male5 + '-idle-up',
+        frames: [{ key: male5, frame: 10 }]
+    })
+
+    scene.anims.create({
+        key: male5 + '-idle-down',
+        frames: [{ key: male5, frame: 1 }]
+    })
+
+    scene.anims.create({
+        key: male5 + '-idle-left',
+        frames: [{ key: male5, frame: 4 }]
+    })
+
+    scene.anims.create({
+        key: male5 + '-idle-right',
+        frames: [{ key: male5, frame: 7 }]
+    })
+    
+    //male 6
+    scene.anims.create({
+        key: male6 + '-idle-up',
+        frames: [{ key: male6, frame: 10 }]
+    })
+
+    scene.anims.create({
+        key: male6 + '-idle-down',
+        frames: [{ key: male6, frame: 1 }]
+    })
+
+    scene.anims.create({
+        key: male6 + '-idle-left',
+        frames: [{ key: male6, frame: 4 }]
+    })
+
+    scene.anims.create({
+        key: male6 + '-idle-right',
+        frames: [{ key: male6, frame: 7 }]
+    })
+    
+    //male 7
+    scene.anims.create({
+        key: male7 + '-idle-up',
+        frames: [{ key: male7, frame: 10 }]
+    })
+
+    scene.anims.create({
+        key: male7 + '-idle-down',
+        frames: [{ key: male7, frame: 1 }]
+    })
+
+    scene.anims.create({
+        key: male7 + '-idle-left',
+        frames: [{ key: male7, frame: 4 }]
+    })
+
+    scene.anims.create({
+        key: male7 + '-idle-right',
+        frames: [{ key: male7, frame: 7 }]
+    })
```

Comment:
This is the tricky one, but first things first - for images of the same structure (which are all in that list) you can put them into array, and iterate through it to create all necessary animations:
```suggestion

    const standardCharacterTextures = ['cat-1', 'cat-2', 'cat-3', 'dog-1', 'dog-2', 'dog-3', 'female05-4', 'female17-1', 'female18-4', 'female19-1', 'female19-3', 'female20-3', 'male3-1', 'male10-1', 'male12-1', 'male13-1', 'male14-2', 'male17-3', 'male17-4']

    standardCharacterTextures.forEach((npc) => {
        scene.anims.create({
            key: npc + '-idle-up',
            frames: [{key: npc, frame: 10}]
        });
        scene.anims.create({
            key: npc + '-idle-down',
            frames: [{key: npc, frame: 1}]
        });
        scene.anims.create({
            key: npc + '-idle-left',
            frames: [{key: npc, frame: 4}]
        });
        scene.anims.create({
            key: npc + '-idle-right',
            frames: [{key: npc, frame: 7}]
        });
    });
```

I wrote that it is a tricky one because generally I am not sure if creating dedicated animation for each character, which is in fact just one frame, is a good solution, but I don't have a better one from the top of my head neither, so for now that should work, I think.